zephyr/lib/dma.c

```diff
@@ -78,12 +78,12 @@ SHARED_DATA struct dma dma[] = {
 	.plat_data = {
 		.dir		= DMA_DIR_DEV_TO_MEM,
 		.caps		= DMA_CAP_HDA,
-#ifdef CONFIG_SOC_INTEL_ACE20_LNL
+#if defined(CONFIG_SOC_INTEL_ACE20_LNL) || defined(CONFIG_SOC_INTEL_ACE30_PTL)
 		.devs		= DMA_DEV_HDA | DMA_DEV_SSP |
```

Comment:
Do we do the same as it is for dai.c or helper.c @marcinszkudlinski @abonislawski